projects/mysadna2/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,17 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
-const routes: Routes = [];
+import { UsersComponent } from './users/users.component';
+import { UserDetailsComponent } from './user-details/user-details.component';
+import { ProjectsComponent } from './projects/projects.component';
+import { ProjectDetailsComponent } from './project-details/project-details.component';
+
+const routes: Routes = [
+  {path:'users', component: UsersComponent}, 
```

Comment:
Which code editor did you decide to use?
You could automatically format your typescript code.
For example VS code on linux do it by `Ctrl + Shift + I`
sublime: `Ctrl + T + F`

Reply:
I'm using VS Code.